src/main/java/jchess/gamelogic/Game.java

```diff
@@ -90,6 +94,7 @@ public void onTimerElapsed() {
 	
 	/**
 	 * This method is called when user makes a click on the Board.
+	 * @param oPosition position the mouse clicked
```

Comment:
Class or Type name should be mentioned instead of variable name.
E.g. @Param IPositionAgent instance for the selected position